tests/unit/test_column_noise.py

```diff
@@ -81,16 +76,16 @@ def test_generate_missing_data(dummy_dataset, default_configuration):
     # Check for expected noise level
     expected_noise = config["row_noise_level"]
     actual_noise = len(newly_missing_idx) / len(orig_non_missing_idx)
-    assert np.isclose(expected_noise, actual_noise, rtol=0.02)
+    assert np.isclose(expected_noise, actual_noise, rtol=0.04)
```

Comment:
This increased a lot with 100k compared to 1mil. Is this acceptable?

Reply:
Yes.